src/GameChooser.tsx

```diff
@@ -0,0 +1,38 @@
+import { ClusterFunGameProps } from './libs';
+import React from 'react';
+import TestatoAssets from 'testLobby/TestGame/assets/Assets';
+
+export interface GameManifestItem { 
+    name: string,
+    displayName?: string, 
+    tags: string[], 
+}
+
+export interface GameDescriptor extends GameManifestItem {
+    logoName: string, 
+    lazyType: React.LazyExoticComponent<any> 
+}
+
+export const allGames: GameDescriptor[] = [
+    {
+        name: "Testato",
+        displayName: "Test Game",
+        tags: [],
+        logoName: TestatoAssets.images.logo,
+        lazyType: React.lazy(() => import('./testLobby/TestGame/views/GameComponent'))
+    }
+]
```

Comment:
A nit about the security model: since all of our games are still found in this list, it will be possible for a player to get games in this list to appear by mocking a manifest that includes it. We should be sure that when we choose to have a game in this list but do not include it in the manifest, it will not be detrimental to the business if someone enables it anyway (i.e. because it behaves badly with the server or has legal/licensing issues).